requirements/docs.txt

```diff
@@ -1,6 +1,6 @@
 # dependencies for building docs, separate from dev.txt as this is also used for builds on readthedocs.org
 # core dependencies
-sphinx>=4.2.0, <6.0.0 # Enforce 4.x sphinx version so that mathjax works correctly
+sphinx>=4.2.0, <5.1.0 # Enforce 4.x sphinx version so that mathjax works correctly
```

Comment:
Can we remove the `# Enforce 4.x sphinx version so that mathjax works correctly` comment at the same time?